### PR TITLE
CONCF-795 | Mercury API — PHP unit tests

### PR DESCRIPTION
**Pull request by Warkot**
:warning: :wrench: WIP, do not merge (yet).

### CODE REVIEW

File: extensions/wikia/MercuryApi/MercuryApiController.class.php [hunk 1]
```diff
@@ -149,6 +149,7 @@ private function getArticleJson( $articleId, Title $title ) {
 	 *
 	 * @param int[] $ids
 	 * @return mixed
+	 * @throws NotFoundApiException
```
Comment (tomasznapieralski) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php]: Exception is catched already

File: extensions/wikia/MercuryApi/models/MercuryApi.class.php [hunk 2]
```diff
@@ -375,6 +375,28 @@ public function processTrendingArticlesData( $data, $paramsToInclude = [] ) {
 		return $items;
 	}
 
+	/**
+	 * @desc To save some bandwidth, the unnecessary params are stripped
+	 *
+	 * @param $item array
```
Comment (tomasznapieralski) [inline extensions/wikia/MercuryApi/models/MercuryApi.class.php]: Type then variable

**Comment by tomasznapieralski** [comment]
CR done, two comments

**Comment by vforge** [comment]
@Warkot please make code mergeable and change PR's description.

**Comment by vforge** [comment]
LGTM

**Pull request merged**